### ISSUE TITLE
Track Preview seek doesn't work consistently (PR submitted)

### ISSUE THREAD

**Issue by jerry924** [LibreTime/libretime#349]
Libretime version: 0dd0a443f

When I click on a track in Tracks library and select Preview, a preview popup window opens up. The song plays fine from the beginning, but as soon as I seek to another part of the file, the seek bar moves position, but the song starts over from the beginning or continues where it was at before I seeked.  Some music files work fine - usually shorter files. I have not narrowed down what makes a file work and not work.

I have tried the following now:

1) I took the jplayer folder content from airtime-mvc/js/jplayer and replaced the entire contents with the jplayer from airtime 2.5.1. Our Airtime 2.5.1 install has no problem with jplayer and seeking. We have been using it for a couple years. With this older version of jplayer and latest Libretime, the same problem happens, so I am pretty sure nothing to do with jplayer.

2) Additionally, when I opened the following path in my browser in another tab which is a direct link to the media path that is playing in jplayer Preview popup:

http://IPADDRESS/api/get-media/file/648

This renders a standard HTML5 player bar where the seek ALSO does not work well for Chrome. It seems like Chrome to Libretime API is not jumping buffer position properly.  

So it seems the problem is the API that is feeding the file into the player is not supporting seek. 

I will keep digging, but I am hoping there are others that might know exactly what is going on here.

**Browsers tested**
Windows 10 Chrome 62.0.3202.94 (Official Build) (64-bit)   - Exhibits issue described above
Windows 10 Firefox 56.0.2 (64-bit) - Exhibits issue described above

**Comment by jerry924** [comment]
I have a shorter music file now that is around 6 minutes long and it also has the same problem. The only way I can jump around the song is to download it and play it locally.   

Has anyone seen this issue before or could this simply be an install/environment problem on my side?

**Comment by jerry924** [comment]
I found the issue by comparing the function smartReadFile($filePath, $size, $mimeType) from Airtime 2.5.1 and latest Libretime.

The following two lines had been removed from Airtime code when moving this code to Libretime. When I put these two lines back, the player works as expected with regard to seek.

`while(ob_get_level() > 0){
   ob_end_flush();
}

$cur = $begin                         //Add this line back into Libretime
fseek($fm,$begin,0);               //Add this line back into Libretime
`

Leaving everything else untouched besides adding these two lines back.
